### PR TITLE
add note about fast recovery during cwnd decrease due to congestion

### LINKED ISSUE #11: Add details about response to ECN marks
It would be nice to describe how a CUBIC sender should respond to ECN marks from the receiver. On receiving an ECE mark, the Linux CUBIC enters a CWR state and follows PRR.

More details about this are available in the document:
https://docs.google.com/document/d/1QiDqBo94wKr1eptTFe6xAB0RWWlwYOy8VLOWrpVAie8/edit

### PR DESCRIPTION
**Pull request by goelvidhi**
As discussed on #11, I added a note about fast recovery.
Please let me know if we need to add more details. IMO, we should avoid mentioning any specific fast recovery mechanism.

Closes #11

### CODE REVIEW

**Review by lisongxu — APPROVED** [review]
Thanks, Vidhi!

**Review by sangtaeha — APPROVED** [review]
Thanks!

**Comment by goelvidhi** [comment]
@larseggert could you please review this PR?

File: draft-eggert-tcpm-rfc8312bis.md [hunk 1]
```diff
@@ -493,9 +493,12 @@ calculated as described in {{win-inc}}.
 ## Multiplicative Decrease {#mult-dec}
 
 When a packet loss is detected by duplicate ACKs or a network
-congestion is detected by ECN-Echo ACKs, CUBIC updates its W_max,
-cwnd, and ssthresh as follows. Parameter beta_cubic SHOULD be set to
-0.7.
+congestion is detected by ECN-Echo ACKs, CUBIC updates its W_max
```
Comment (larseggert) [inline draft-eggert-tcpm-rfc8312bis.md:496]:
```suggestion
congestion is detected by receiving packets marked with ECN-Echo (ECE), CUBIC updates its W_max
```

File: draft-eggert-tcpm-rfc8312bis.md [hunk 2]
```diff
@@ -493,9 +493,12 @@ calculated as described in {{win-inc}}.
...
+and reduces its cwnd and ssthresh immediately as below. For both
+packet loss and congestion detection by ECN, the sender MAY
```
Comment (larseggert) [inline draft-eggert-tcpm-rfc8312bis.md:498]:
```suggestion
packet loss and congestion detection through ECN, the sender MAY
```

**Review by larseggert — CHANGES_REQUESTED** [review]

**Review by larseggert — APPROVED** [review]

**Pull request merged**